meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio-mfgtool/imx6ullevk/lmp.cfg

```diff
@@ -51,7 +51,7 @@ CONFIG_OPTEE=y
 CONFIG_OF_LIBFDT_OVERLAY=y
 CONFIG_OPTEE_TZDRAM_SIZE=0x02000000
 CONFIG_OPTEE_TZDRAM_BASE=0x9E000000
-CONFIG_OPTEE_SKIP_LOWLEVEL_INIT=y
+CONFIG_SKIP_LOWLEVEL_INIT=y
```

Comment:
just asking... do we need the same for meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio-mfgtool/imx6ulevk/lmp.cfg?

Reply:
@angolini this set is for 6ull only. 6ul support is coming with another patchset.

Reply:
ok